NewSpring/ApollosProject
# [DEV-3777] Creates API for interactions

## DESCRIPTION

### What does this PR do, or why is it needed?

Creates the API work needed to create interactions in Rock when someone prays.

### How do I test this PR?

1) Yarn Start
2) Go to your localhost:4000 and run a `incrementPrayed` on a prayer like `PrayerRequest:ab914f6fe3271e951cde1ef1deb1736d`.
3) Go to https://alpha-rock.newspring.cc/page/147 and run:
```
SELECT
    TOP 10 *
FROM
    [Interaction]
ORDER BY
    InteractionDateTime DESC
```
Then look to see if your interaction shows up.

---

> I am affirming this is my _best_ work ([Ecclesiastes 9:10](https://www.bible.com/bible/97/ECC.9.10.MSG))

## TODO


## REVIEW

**Manual QA**


**Code Review: Questions to consider**


---

> The purpose of PR Review is to _improve the quality of the software._
